validate.js

```diff
@@ -0,0 +1,28 @@
+function validateMail() {
+  const form = document.getElementById('myForm');
+  const email = form.elements.email.value;
```

Comment:
- [ ] If a user submits the contact form with the email field containing only lowercase characters the form should be sent normally, but it still shows an error after I correct the uppercase letter into a lowercase. 
<img width="625" alt="Screenshot 2022-03-08 at 17 50 40" src="https://user-images.githubusercontent.com/77038610/157251005-adafbe73-99f7-4266-af4d-4a206958eda2.png">

- [ ] OPTIONAL: I recommend you styling the error message and making it more user friendly. Currently it looks confusing, as you can see ✌🏻 
<img width="669" alt="Screenshot 2022-03-08 at 17 52 41" src="https://user-images.githubusercontent.com/77038610/157251354-265392f1-76c0-4476-a640-025fd81b7481.png">

Reply:
Hello, how do i solve this